mojaloop/values.yaml

```diff
@@ -6001,6 +6001,637 @@ central:
+                      "debug": "all",
+                      "metadata.broker.list": "$kafka_host:$kafka_port",
+                      "client.id": "cep-prod-notification-event",
+                      "event_cb": true,
+                      "compression.codec": "none",
+                      "retry.backoff.ms": 100,
+                      "message.send.max.retries": 2,
+                      "socket.keepalive.enable": true,
+                      "queue.buffering.max.messages": 10000000,
+                      "batch.num.messages": 100,
+                      "dr_cb": true,
+                      "socket.blocking.max.ms": 1,
+                      "queue.buffering.max.ms": 1,
+                      "broker.version.fallback": "0.10.1.0",
+                      "api.version.request": true
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "emailSettings": {
+            "smtpConfig": {
+              "host": "$email_host",
+              "port": $email_port,
+              "secureConnection": false,
+              "tls": {
+                "ciphers": "$email_tls_ciphers"
+            },
+              "auth": {
+                "user": "$email_user",
+                "pass": "$email_pass"
+              }
+            }
+          },
+          "healthCheckPort": $port
+        }
+
+    init:
+      enabled: true
+
+    service:
+      name: email-notifier
+      type: ClusterIP
+      externalPort: 3081
+      internalPort: 3081
+
+      annotations: {}
+
+      # This allows one to point the service to an external backend. 
+      # This is useful for local development where one wishes to hijack 
+      # the communication from the service to the node layer and point
+      # to a specific endpoint (IP, Port, etc).
+      external:
+        enabled: false
+        # 10.0.2.2 is the magic IP for the host on virtualbox's network
+        ip: 10.0.2.2
+        ports:
+          api: 
+            name: email-notifier
+            externalPort: 3081
+            internalPort: 3081
+
+    readinessProbe:
+      enabled: true
+      httpGet:
+        path: /healthcheck
+      initialDelaySeconds: 30
+      periodSeconds: 15
+
+    livenessProbe:
+      enabled: true
+      httpGet:
+        path: /healthcheck
+      initialDelaySeconds: 30
+      periodSeconds: 15
+
+    ingress:
+      enabled: true
+      # Used to create an Ingress record.
+      hosts:
+        api: email-notifier.local
+      
+      externalPath: /
+      
+      annotations:
+        # kubernetes.io/ingress.class: nginx
+        # kubernetes.io/tls-acme: "true"
+      
+      tls:
+        # Secrets must be manually created in the namespace.
+        # - secretName: chart-example-tls
+        #   hosts:
+        #     - chart-example.local
+
+    resources: {}
+  centraleventprocessor:
+    replicaCount: 1
+    image:
+      repository: mojaloop/central-event-processor
+      tag: v0.1
```

Comment:
This should match the sprint release.

I believe we are in 4.1.0?

@elnyry please confirm confirm.